DependencyInjection/DoctrineExtension.php

```diff
@@ -493,6 +495,11 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('orm.xml');
 
+        if ((new ReflectionMethod(EntityManager::class, '__construct'))->isPublic()) {
```

Comment:
Should there be a negation here? This seems to do the opposite of what your commit message says.

```suggestion
        if (! (new ReflectionMethod(EntityManager::class, '__construct'))->isPublic()) {
```

Reply:
Indeed, fixed!